diff --git a/node_modules/react-native-splash-screen/ios/RNSplashScreen.m b/node_modules/react-native-splash-screen/ios/RNSplashScreen.m
index 02bf3e5..ce92416 100644
--- a/node_modules/react-native-splash-screen/ios/RNSplashScreen.m
+++ b/node_modules/react-native-splash-screen/ios/RNSplashScreen.m
@@ -51,7 +51,11 @@ + (void)hide {
         });
     } else {
         dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.1 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
-            [loadingView removeFromSuperview];
+            [UIView animateWithDuration:0.4
+                    delay:0
+                    options:UIViewAnimationOptionCurveEaseIn
+                animations:^{loadingView.alpha = 0.0;}
+                completion:^(BOOL finished){ [loadingView removeFromSuperview]; }];
         });
     }
 }
